akka-stream/src/main/scala/akka/stream/scaladsl/StreamConverters.scala

```diff
@@ -90,4 +96,91 @@ object StreamConverters {
   def asInputStream(readTimeout: FiniteDuration = 5.seconds): Sink[ByteString, InputStream] =
     Sink.fromGraph(new InputStreamSinkStage(readTimeout))
 
+  /**
+    * A `Sink` that will use Java 8 `Collector` to reduce and optionally transform
+    * accumulated result into a final representation.
+    * As collector is mutable you can use it only with one stream.
+    * Reduction operations perform sequentially.
+    */
```

Comment:
The docs here feels a bit sparse compared to the existing methods, is that because there are no more important things to describe about these methods or could it be improved?